src/components/semantic/metaItems.tsx

```diff
@@ -72,8 +73,24 @@ export const MetaItems = asMetaItems({
     emailEventDetails: ["Email Event Details", {type: "textarea"}],
     emailConfirmedBookingText: ["Email Confirmed Booking Text", {type: "textarea"}],
     emailWaitingListBookingText: ["Email Waiting List Booking Text", {type: "textarea"}],
-    date: ["Start Date", {presenter: DateTimeInput}],
-    end_date: ["End Date", {presenter: DateTimeInput}],
+    date: ["Start Date", {
+        presenter: DateTimeInput,
+        hasWarning: (value) => {
+            if (!value) {
+                return "Start Date must be populated";
+            }
+        },
+        deleteIfEmpty: true,
+    }],
+    end_date: ["End Date", {
+        presenter: DateTimeInput,
+        hasWarning: (value) => {
+            if (!value) {
+                return "End Date must be populated";
+            }
+        },
+        deleteIfEmpty: true,
```

Comment:
Instead of deleting if empty, can we not force it to be the same as the start date, if that's going to be a problem for the content model?

Reply:
Connecting the values in these two metadata items wouldn't be easy at all because of how the editor is built. 
The only way to do it that I can see currently would be super hacky and might break other date fields in the editor.
Maybe a different way of fixing it is stopping the page being published if there are any warnings?

Reply:
After taking another look I might have an idea...

Reply:
To be honest @Morpheu5, I don't like the idea of deciding an arbitrary end date for events based on the start date. Either content choose it explicitly or the event is broken imo.

Reply:
@chrisjpurdy fair enough, I'm OK with that too, as long as they are aware. Can we make it a content error instead?

Reply:
If we _really_ want to do this though, I've got a clean-ish solution

Reply:
I mean, from a UX perspective, it's probably a good idea, it saves them an extra edit step in case they forget, and if they need to change the end date, they can still do it…

Reply:
I think for now the content error is the right thing to do, it still feels a bit off setting the end date arbitrarily 🤷 I'll make another card for that since it's an API change.
Maybe we need to have something in the editor that warns you of any content errors before publishing a document, that would at least stop the content team having to:
 - Wait for ETL
 - Go to the content errors page
 - Find the relevant error